app/_src/policies/meshexternalservice.md

```diff
@@ -0,0 +1,317 @@
+---
+title: MeshExternalService
+---
+
+{% warning %}
+Do **not** combine with [External Service](/docs/{{ page.version }}/policies/external-services).
```

Comment:
what happens then? How should I migrate from ExternalService then if I can't combine it?

Reply:
I need to check what happens if you create a hostname generator that overshadows a domain hijacked by regular ExternalService.

Reply:
Added a note in DNS setup section to not hijack real DNS addresses.